infra/notifications/templates/messageTemplates.js

```diff
@@ -127,6 +133,15 @@ const messageTemplates = {
         text: template(
           fs.readFileSync(`${templateDir}/seller-OfferFinalized.txt`).toString()
         )
+      },
+      OfferData: {
+        subject: template('New Review for <%= listing.title %>'),
+        mjml: template(
+          fs.readFileSync(`${templateDir}/buyer-OfferReview.mjml`).toString()
```

Comment:
These files should get renamed to the seller side
```suggestion
          fs.readFileSync(`${templateDir}/seller-OfferReview.mjml`).toString()
```

Reply:
Yeah, that's reasonable. I'd suggest to change it too.

Reply:
I don't see the template files seller-OfferReview.mjml/txt
They should be added as part of this PR.